docs/environment.rst

```diff
@@ -201,6 +201,11 @@ These environment variables are optional:
 
   If you are not sure what this means leave it unset.
 
+* **DROP_CLIENT**
+
+  Disconnect any client that matches this regular expression. This is useful
+  for disconnecting buggy clients.
+
```

Comment:
Please update this to match the surrounding doc format (you're using the old style).  Also be more specific about what exactly is being matched, and perhaps give an example value.